modules/obj/test/obj-loader.spec.js

```diff
@@ -76,6 +77,19 @@ test('OBJLoader#parseText - multi-part object', async (t) => {
   t.end();
 });
 
+test('OBJLoader#parseText - object with vertex colors', async (t) => {
+  const data = await load(OBJ_VERTEX_COLOR_URL, OBJLoader);
+  validateMeshCategoryData(t, data);
+
+  t.equal(data.attributes.POSITION.value.length, 108, 'POSITION attribute was found');
+  t.equal(data.attributes.POSITION.size, 3, 'POSITION attribute was found');
+  t.equal(data.attributes.NORMAL.value.length, 108, 'NORMAL attribute was found');
+  t.equal(data.attributes.NORMAL.size, 3, 'NORMAL attribute was found');
+  t.equal(data.attributes.COLOR_0.value.length, 108, 'COLOR_0 attribute was found');
+  t.equal(data.attributes.COLOR_0.size, 3, 'COLOR_0 attribute was found');
```

Comment:
Nit: Might want to check one of the vertex values?

Reply:
Is there a non-flappy way to test float values? I'm getting stuff like
```
expected: 0.2801
actual:   0.2800999879837036
```

Reply:
I see deck using `import {equals} from '@math.gl/core';` so I'll try that